src/commands/request/system/skip-request-data-silos/impl.ts

```diff
@@ -0,0 +1,30 @@
+import type { LocalContext } from '../../../../context';
+import type { RequestStatus } from '@transcend-io/privacy-types';
+import { skipRequestDataSilos as skipRequestDataSilosHelper } from '../../../../lib/requests';
+
+interface SkipRequestDataSilosCommandFlags {
```

Comment:
whats the thinking around having these files separate? why not just import `skipRequestDataSilosHelper` into the command file?